### PR TITLE
ui components and wireframe

### PR DESCRIPTION
**Pull request by RiccLQL**

### CODE REVIEW

**Comment by RiccLQL** [comment]
@slaroche

File: ui/src/components/library/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  useEffect(() => {
+    switch (props.size) {
+    case ButtonSizes.small:
+      setStyle({ width: '50px', height: '20px', backgroundColor: props.color })
+      break
+    case ButtonSizes.medium:
+      setStyle({ width: '80px', height: '30px', backgroundColor: props.color })
+      break
+    }
+  }, [])
```
Comment (slaroche) [inline ui/src/components/library/Button.tsx:23]: im not sure but I think you need to add props.size  here

File: ui/src/components/library/Enums.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export enum ButtonSizes{
+  small = 'small',
+  medium = 'medium',
+}
+
+export enum ButtonColors{
+  green = '#54E095',
+  blue = '#547BE0',
+}
```
Comment (slaroche) [inline ui/src/components/library/Enums.tsx]: new line

File: ui/src/components/library/LogItem.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { QueryStreamValue } from 'services/Client'
+// import { LogLabel } from './LogLabel'
```
Comment (slaroche) [inline ui/src/components/library/LogItem.tsx]: rm

File: ui/src/components/library/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  const [style, setStyle] = useState<React.CSSProperties>({})
+
+  useEffect(() => {
+    switch (props.size) {
+    case ButtonSizes.small:
+      setStyle({ width: '50px', height: '20px', backgroundColor: props.color })
+      break
+    case ButtonSizes.medium:
+      setStyle({ width: '80px', height: '30px', backgroundColor: props.color })
```
Comment (slaroche) [inline ui/src/components/library/Button.tsx]: you should use `rem` instead of `px`!

File: ui/src/components/library/LogLabel.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+interface Props {
+  label: string,
+  value: string,
+}
+
+export const LogLabel = (props: Props) => {
```
Comment (slaroche) [inline ui/src/components/library/LogLabel.tsx]: : JSX.Element

File: ui/src/components/Query.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { InputBar } from 'components/library/InputBar'
+import { ButtonColors, ButtonSizes, InputTypes } from './library/Enums'
```
Comment (slaroche) [inline ui/src/components/Query.tsx]: please use fullpath or relative path, not both.


nit: I would prefered full path

**Pull request merged**